nextcord/gateway.py

```diff
@@ -420,13 +420,7 @@ async def identify(self):
             "op": self.IDENTIFY,
             "d": {
                 "token": self.token,
-                "properties": {
-                    "$os": sys.platform,
-                    "$browser": "nextcord",
-                    "$device": "nextcord",
-                    "$referrer": "",
-                    "$referring_domain": "",
-                },
+                "properties": {"os": sys.platform, "browser": "nextcord", "device": "nextcord"},
```

Comment:
I'd recommend to keep the comma after the device name so it displays on separate lines

```suggestion
                "properties": {
                    "os": sys.platform,
                    "browser": "nextcord",
                    "device": "nextcord",
                },
```